bin/cylc-help

```diff
@@ -292,6 +292,7 @@ admin_commands['profile-battery'] = ['profile-battery']
 admin_commands['import-examples'] = ['import-examples']
 admin_commands['upgrade-run-dir'] = ['upgrade-run-dir']
 admin_commands['check-software'] = ['check-software']
+admin_commands['wrap-func'] = ['wrap-func']
```

Comment:
More like `call-func` than `wrap-func`?

Reply:
Well, "call" suggests a normal function call, doesn't it?  This wraps a function into a minimal executable program, to execute in the process pool.

Reply:
Changed to `function-run` - less ambiguous, an no abbrev similarity to other commands (`cylc warranty` was a problem with `wrap-func`; and `cylc run-func` is a no-go thanks to `cylc run`).